src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.js

```diff
@@ -289,12 +298,13 @@ class ActivityFeed extends React.Component<Props, State> {
 
         return (
             // eslint-disable-next-line
-            <div className="bcs-activity-feed" data-testid="activityfeed" onKeyDown={this.onKeyDown}>
+            <div className={classNames("bcs-activity-feed", { "bcs-is-scrolled": this.state.isScrolled })} data-testid="activityfeed" onKeyDown={this.onKeyDown}>
```

Comment:
Can we destructure `isScrolled` from state above? We already do so for `isInputOpen`.